### ISSUE TITLE
[1.7.10] BiomeTweaker does not create scripts folder and mod is "loaded" not not "active"

### ISSUE THREAD

**Issue by Arkadiumx** [superckl/BiomeTweaker#300]
I cannot seem to get the mod to work for the latest Forge build 10.13.4.1614 for 1.7.10. It doesn't crash or anything but once forge loads the menu says "5 mods loaded 4 mods active" and the scripts folder is missing from the BiomeTweaker folder in the configs directory. I've tried multiple builds of BiomeTweaker with the same result. Am I doing something wrong?

**Comment by Dinerell** [comment]
In 1.7 and 1.12 it wasn't creating a dedicated scripts folder. There should be an example file in the configs.

**Comment by Arkadiumx** [comment]
> In 1.7 and 1.12 it wasn't creating a dedicated scripts folder. There should be an example file in the configs.

Yes I noticed this behavior but even after manually creating a scripts folder it acts as if there are no scripts.

**Comment by Dinerell** [comment]
> Yes I noticed this behavior but even after manually creating a scripts folder it acts as if there are no scripts.

You misunderstood me there a bit. You don't need a scripts folder, you link your scripts path in the cfg-File or you just put it inside the config folder **config/BiomeTweaker**.
https://github.com/superckl/BiomeTweaker/wiki/Main-Configuration-File

**Comment by Arkadiumx** [comment]
> You misunderstood me there a bit. You don't need a scripts folder, you link your scripts path in the cfg-File or you just put it inside the config folder **config/BiomeTweaker**. https://github.com/superckl/BiomeTweaker/wiki/Main-Configuration-File

Gotcha, I will try this. Thank you for answering my 6 month old post lol.
